src/Draco.Editor.Web/app/logo-handler.js

```diff
@@ -0,0 +1,48 @@
+import {get} from 'https';
+
+export function CreateThemeBasedLogo() {
+    var logoLight = "";
+    get("https://raw.githubusercontent.com/Draco-lang/Language-suggestions/main/Resources/Logo-Short.svg", (response) => logoLight = response);
+    var lightLines = logoLight.split("\n");
+    lightLines.splice(0, 1);
```

Comment:
Here you have 2 strictly identical behavior, a method would be good, and would improve code readability, even without comments.